msteams-platform/sbs-gs-notificationbot.yml

```diff
@@ -248,7 +247,7 @@ items:
   content: |
```

Comment:
The "**Take tour of source code**" section above (start from line 218) is out of date. The latest (v5) folder structure is different. Please see [here](https://github.com/OfficeDev/TeamsFx/blob/main/templates/scenarios/js/notification-restify/README.md#whats-included-in-the-template) for the latest source code structure.

Reply:
And here's a sample of the screenshot.
<img width="197" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/7642967/3d47ea7e-808a-4779-b064-85feca2ec2bc">

Reply:
Also remove line 223: ~"If you choose to use UI framework, following sample template code provides a scaffolding with React components."~

Reply:
Updated as suggested.